StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetError.swift

```diff
@@ -69,73 +69,128 @@ extension PaymentSheetError: CustomDebugStringConvertible {
+                return "Invalid client secret"
+            case .accountLinkFailure:
+                return STPLocalizedString(
+                    "Something went wrong when linking your account.\nPlease try again later.",
+                    "Error message when an error case happens when linking your account"
+                )
+            case .paymentIntentInTerminalState(status: let status):
+                return "PaymentSheet received a PaymentIntent in a terminal state: \(status)"
+            case .setupIntentInTerminalState(status: let status):
+                return "PaymentSheet received a SetupIntent in a terminal state: \(status)"
+            case .fetchPaymentMethodsFailure:
+                return "Failed to retrieve PaymentMethods for the customer"
+            case .linkSignUpNotRequired:
+                return "Don't call sign up if not needed"
+            case .noPaymentMethodTypesAvailable(intentPaymentMethods: let intentPaymentMethods):
+                return "None of the payment methods on the PaymentIntent/SetupIntent can be used in PaymentSheet: \(intentPaymentMethods). You may need to set `allowsDelayedPaymentMethods` or `allowsPaymentMethodsRequiringShippingAddress` in your PaymentSheet.Configuration object."
+            case .linkCallVerifyNotRequired:
+                return "Don't call verify if not needed"
+            case .linkingWithoutValidSession:
+                return "Linking account session without valid consumer session"
+            case .savingWithoutValidLinkSession:
+                return "Saving to Link without valid session"
+            case .payingWithoutValidLinkSession:
+                return "Paying with Link without valid session"
+            case .deletingWithoutValidLinkSession:
+                return "Deleting Link payment details without valid session"
+            case .updatingWithoutValidLinkSession:
+                return "Updating Link payment details without valid session"
+            case .failedToCreateLinkSession:
+                return "Failed to create Link account session"
+            case .linkNotAuthorized:
+                return "confirm called without authorizing Link"
+            case .setupIntentClientSecretProviderNil:
+                return "setupIntentClientSecretForCustomerAttach, but setupIntentClientSecretProvider is nil"
+            }
+        }()
+
         switch self {
-        case .unknown(debugDescription: let message):
-            return "An unknown error occurred in PaymentSheet. " + message
-        case .linkLookupNotFound(serverErrorMessage: let message):
-            return "An error occurred in PaymentSheet. " + message
+        case .unknown:
+            return "An unknown error occurred in PaymentSheet. " + errorMsesageSuffix
         default:
-            return "An error occurred in PaymentSheet. " + safeLoggingString
+            return "An error occurred in PaymentSheet. " + errorMsesageSuffix
```

Comment:
change errorMsesageSuffix to errorMessageSuffix

Reply:
good catch, fixed!